### PR TITLE
Benchmarks: micro benchmarks - add python code for DirectXGPUCopy

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
add python code for DirectXGPUCopy.

### CODE REVIEW

File: superbench/benchmarks/micro_benchmarks/directx_gpu_copy_performance.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+        self._mem_types = ['htod', 'dtoh']
+        self._bin_name = 'GPUCopyBw.exe'
```
Comment (cp5555) [inline superbench/benchmarks/micro_benchmarks/directx_gpu_copy_performance.py:23]: Also the same question as Ziyue: The name is not related to this benchmark.

**Review by cp5555 — APPROVED** [review]

File: superbench/benchmarks/micro_benchmarks/directx_gpu_copy_performance.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            type=int,
+            required=False,
+            default=20,
+            help='Number of warm up copy times to run.',
+        )
+        self._parser.add_argument(
+            '--num_loops',
+            type=int,
+            required=False,
+            default=100000,
```
Comment (yzygitzh) [inline superbench/benchmarks/micro_benchmarks/directx_gpu_copy_performance.py:47]: Why do we need such huge num_loops? For gpu_copy I think it's around 1k.

**Review by yzygitzh — APPROVED** [review]

File: superbench/benchmarks/micro_benchmarks/directx_gpu_copy_performance.py (hunk 1 above)
Comment (yukirora) [reply superbench/benchmarks/micro_benchmarks/directx_gpu_copy_performance.py]: renamed to DirectXGPUCopyBw.exe

File: superbench/benchmarks/micro_benchmarks/directx_gpu_copy_performance.py (hunk 2 above)
Comment (yukirora) [reply superbench/benchmarks/micro_benchmarks/directx_gpu_copy_performance.py]: updated

**Pull request merged**